trace-sdk/build.gradle

```diff
@@ -81,4 +82,16 @@ junitJacoco {
     jacocoVersion = '0.8.6'
 }
 
+def getTraceToken() {
+    // try to get the environment variable in CI
+    def traceToken = findProperty("traceToken")
+    if (traceToken != null) {
+        return traceToken
+    }
+    // if we don't have a env var - get it from the local.properties
+    Properties properties = new Properties()
+    properties.load(project.rootProject.file('local.properties').newDataInputStream())
+    return properties.getProperty("traceToken");
```

Comment:
Shouldn't we throw some error/exception, if this returns null?

Reply:
that's a good question - i figured it was okay to silently fail as it's only used in one place and it's likely most people aren't going to use it. If they run the test that needs it - the tests would fail. Perhaps it makes sense to add a note to those tests to make sure people have it set?